lib/generators.nix

```diff
@@ -175,4 +175,51 @@ rec {
                          else "<λ:{${showFnas}}>"
     else abort "toPretty: should never happen (v = ${v})";
 
+  # PLIST handling
+  toPLIST = {}: v: let
```

Comment:
I’m torn. When I search for it, everyone else doesn’t write `PLIST`, but `plist`. So I’d prefer camel case in this case.